heartbeat/monitors/active/http/http.go

```diff
@@ -71,7 +71,7 @@ func create(
 		}
 	} else {
 		for i, url := range config.URLs {
-			jobs[i], err = newHTTPMonitorIPsJob(&config, url, tls, enc, body, validator)
+			jobs[i], err = newHTTPMonitorIPsJob(url, &config, tls, enc, body, validator)
```

Comment:
Is there a reason you changed the params order?

Reply:
I changed it to be consistent with the param order of `newHTTPMonitorHostJob`. Can revert it if needed.